libraries/AP_HAL_Linux/AnalogIn_ADS1115.cpp

```diff
@@ -62,7 +62,7 @@ AP_HAL::AnalogSource* AnalogIn_ADS1115::channel(int16_t pin)
 
 void AnalogIn_ADS1115::init()
 {
-    _adc->init();
+    _adc->init(0);
```

Comment:
I really don't like the hard-coded i2c bus here. Either add a specific #define somewhere, or after you overload that init() function, just revert this line back to what it was.